src/module/dice/die-pools.mjs

```diff
@@ -0,0 +1,24 @@
+export default class GrimwildDiePoolRoll extends Roll {
+	static CHAT_TEMPLATE = "systems/grimwild/templates/chat/die-pool-action.hbs";
+
+	async render({flavor, template=this.constructor.CHAT_TEMPLATE, isPrivate=false}={}) {
+		if (!this._evaluated) await this.evaluate();
+
+		const chatData = {
+			formula: isPrivate ? "???" : this._formula,
+			flavor: isPrivate ? null : flavor ?? this.options.flavor,
+			user: game.user.id,
+			tooltip: isPrivate ? "" : await this.getTooltip(),
+			stat: this.options.stat,
+			dice: this.dice[0].results,
+            startPool: isPrivate ? "???" : "",
+            endPool: isPrivate ? "???" : "",
```

Comment:
Looks like a discrepancy in your editor here, the JS files are configured to use tabs rather than spaces. You should be able to resolve that with `npm run lint:fix`, and I've also pushed up a new commit to `main` that will fix the existing errors/warnings barring a few that could have caused conflicts.